Add info on using the summary instruments toggle (#5)
dynatrace-oss-contrib/spring-boot: spring-boot-project/spring-boot-docs/src/docs/asciidoc/actuator/metrics.adoc

* Enrich with Dynatrace metadata: If a OneAgent or Dynatrace operator is running, enrich metrics with additional metadata (for example, about the host, process, or pod).
* Default dimensions: Specify key-value pairs that are added to all exported metrics.
If tags with the same key are specified with Micrometer, they overwrite the default dimensions.

> * Use Dynatrace Summary instruments: In some cases the Micrometer Dynatrace registry created metrics that were rejected. Since Micrometer 1.9.x, this should be fixed. Setting this toggle to false forces Micrometer to fall back to the behavior that was the default before 1.9.x. It should only be used if encountering problems while migrating from 1.8.x to 1.9.x.

Reviewer: ```suggestion
* Use Dynatrace Summary instruments: In some cases the Micrometer Dynatrace registry created metrics that were rejected. In Micrometer 1.9.x, this was fixed by introducing Dynatrace-specific summary instruments. Setting this toggle to false forces Micrometer to fall back to the behavior that was the default before 1.9.x. It should only be used when encountering problems while migrating from 1.8.x to 1.9.x.
```

	          default-dimensions:
	            key1: "value1"
	            key2: "value2"

> 	          use-dynatrace-summary-instruments: true

Reviewer: ```suggestion
	          use-dynatrace-summary-instruments: true # (default: true)
```
